hapi-plugin/lib/plugin.js

```diff
@@ -17,9 +17,13 @@ const optionsSchema = joi.object().keys({
 	bucket: joi.string().alphanum(),
 	countSuccess: joi.boolean().default(true),
 	addHeaders: joi.boolean().default(true),
+	headerLimit: joi.string().default('X-RateLimit-Limit'),
+	headerRemaining: joi.string().default('X-RateLimit-Remaining'),
+	headerReset: joi.string().default('X-RateLimit-Reset'),
 	message: joi.string().default('you have exceeded your request limit'),
 	onError: joi.func().arity(3),
 	getKey: joi.func().arity(1).default(getRequestIP),
+	ttlTransform: joi.func().arity(1).default(identityTTL),
```

Comment:
I wonder if it might be better to just use an arrow func here -
`ttlTransform: joi.func().arity(1).default(ttl => ttl)`